airflow/www/gunicorn_config.py

```diff
@@ -30,3 +30,10 @@ def post_worker_init(_):
     """
     old_title = setproctitle.getproctitle()
     setproctitle.setproctitle(settings.GUNICORN_WORKER_READY_PREFIX + old_title)
+
+
+def on_starting(server):
```

Comment:
This is the server hook - https://docs.gunicorn.org/en/stable/settings.html#on-starting

(Note for future me and readers :D )